Sheepish-Squirrels/bangazon-sprint-2
# Prompt functionality for post new product

## Changes
- Created tests for getProduct
- Formatted payment-type test file
- Created prompt functionality for posting new product

## Testing
1. Fetch vf-post-product
2. `npm test`
3. run `node app/cli.js`
4. Choose Option 4. You should get a message telling you to choose an active customer first and then it displays the menu again.
5. Choose Option 2. Enter in a number for the customer id
6. Choose Option 4. Enter in data for product. Once you complete, app should display menu.
7. Open db browser. You should see the new product with customer_id of the active customer you chose.

## Expected Result
- All tests should pass
- User can't post a new product without choosing an active customer first
- Product should post to database with customer_id equal to the chosen active customer id
- App should display menu when complete or after displaying the error

## Tickets
#4
